src/MessagePane.tsx

```diff
@@ -58,20 +62,28 @@ class SuggestedActions extends React.Component<MessagePaneProps, {}> {
 
 }
 
-function suggestedActions(activities: Activity[]) {
+function activityWithSuggestedActions(activities: Activity[]) {
```

Comment:
for consistency, let's either call this `activityWithActions` or call the property `activityWithSuggestedActions`.